gpflow/base.py

```diff
@@ -113,7 +104,7 @@ class PriorOn(Enum):
 class Parameter(tfp.util.TransformedVariable):
     def __init__(
         self,
-        value: Union[TensorData, "Parameter"],
+        value: "TensorData",
```

Comment:
> Was there a reason why we previously had a `Union` with `Parameter` and we no longer do? I'm guessing we don't need it because `TensorData` has `Parameter` within its `TensorType` element, but thought I should check.

Exactly. `Parameter` is already a part of `TensorData`. I assume the `Union` is just some legacy cruft...